en/02_Developer_Guides/03_Forms/Field_types/04_GridField.md

```diff
@@ -214,18 +212,16 @@ $gridField->setConfig($config);
 
 ### `GridFieldConfig_RecordViewer`
 
+> [!CAUTION]
+> The `DataObject` class displayed must define a `canView()` method that returns a boolean on whether the user can view
+> this record.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.